### PR TITLE
Add loginErrorDisplayConfig prop to AuthUIContextProvider

### PR DESCRIPTION
**Pull request by daileytj**
Fixes PXBLUE-2313.

#### Changes proposed in this Pull Request:
- Add loginErrorDisplayConfig prop to AuthUIContextProvider

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -57,7 +57,10 @@ import { AuthUIContextProvider } from '@pxblue/react-native-auth-workflow';
     -   Default: Shows the `projectImage`
 -   **loginType** (optional): _`'email' | 'username'`_
     -   Type of input to use for login. (Currently available for use with @pxblue/react-auth-workflow@2.2.0 and above)
-    -   Default: "email
+    -   Default: 'email'
+-   **loginErrorDisplayConfig** (optional): _`LoginErrorDisplayConfig`_
+    -   Custom configuration for the error message display on the login screen.
+    -   Default: {mode: dialog}
```
Comment (joebochill) [inline docs/API.md:63]:
```suggestion
    -   Default: { mode: 'dialog' }
```

File: docs/API.md [hunk 2]
```diff
@@ -344,6 +347,28 @@ Network state and email for a user attempting to authenticate into the app. Exte
...
+## LoginErrorDisplayConfig
+
+Custom configuration for the error message display on the login screen.
+
+### Type Declaration
+
+-   **mode** (optional): _`'dialog' | 'message-box' | 'both' | 'none'`_
+    -   Sets whether to show login error messages via a pop-up dialog, an inline message box, both, or neither.
+    -   Default: 'dialog'
+-   **dismissible** (optional): _`boolean`_
```
Comment (joebochill) [inline docs/API.md:359]: I just went down the rabbit hole on this...I feel like I always spell this with an A but that is apparently not correct.

**Review by joebochill — APPROVED** [review]

**Comment by joebochill** [comment]
Out of curiosity and because I don't remember, does RN also show a popup dialog for errors as of now?

File: docs/API.md (hunk 2 above)
Comment (daileytj) [reply docs/API.md:359]: If I didn't have a spellchecker plugin in VS code It would have been spelled with an A.

**Pull request merged**